src/components/Toolbar.jsx

```diff
@@ -1,33 +1,71 @@
-import React from 'react'
+import React, { Component } from 'react'
 import { PropTypes } from 'prop-types'
 import { Button, Icon } from 'cozy-ui/react'
-import IconTeam from '../assets/icons/team.svg'
+
 import { translate } from 'cozy-ui/react/I18n'
+import { Query } from 'cozy-client'
+
+import ContactFormModal from './Modals/ContactFormModal'
+import withModal from './HOCs/withModal'
+import ContactCardModalConnected from './Modals/ContactCardModalConnected'
```

Comment:
the file `ContactCardModalConnected ` has been deleted in this PR..?

Reply:
Yep! But you're commenting an 'Outdated' code. :D